features/exploredash/src/main/res/layout/buy_gift_card_description.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/root_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/collapse_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_marginTop="15dp"
+        android:layout_marginRight="15dp"
+        app:srcCompat="@drawable/ic_top_navigation_close_button"
+        />
+
+    <TextView
+        android:id="@+id/buy_card_header_label"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="@id/left_guideline"
+        app:layout_constraintEnd_toEndOf="@id/right_guideline"
+        app:layout_constraintTop_toBottomOf="@id/collapse_button"
+        android:layout_marginTop="40dp"
+        android:textSize="28sp"
+        android:gravity="center"
+        android:text="@string/buy_card_header_text"
+        android:fontFamily="@font/montserrat_bold"
+        android:textColor="@color/gray_900"
+        android:paddingBottom="2dp"
+        android:paddingLeft="8dp"
+        android:paddingRight="8dp"
+        />
+
+    <TextView
+        android:id="@+id/buy_card_desc_label"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="@id/left_guideline"
+        app:layout_constraintEnd_toEndOf="@id/right_guideline"
+        app:layout_constraintTop_toBottomOf="@id/buy_card_header_label"
+        android:gravity="left|center"
+        android:text="@string/buy_card_desc_text"
+        android:fontFamily="@font/montserrat_regular"
```

Comment:
Please consider using text styles from the common module. I've created some styles that adhere to our [typography design system](https://www.figma.com/file/azdJACb5WmivxYVhB5q46F/Android-%7C-Design-system?node-id=479%3A147) so that we have it unified across the app eventually. This text should be `Body2` style. Not all styles are implemented, so add yours as needed.